corehq/apps/cloudcare/static/cloudcare/js/form_entry/entries.js

```diff
@@ -388,6 +388,21 @@ hqDefine("cloudcare/js/form_entry/entries", function () {
         self.templateType = 'select';
         self.choices = question.choices;
         self.isMulti = true;
+        self.hideLabel = ko.observable(options.hideLabel);
+
+        // A custom binding to add a column layout class only if hideLabel = true
+        ko.bindingHandlers.addColStyle = {
```

Comment:
I think this binding should live in a different part of the code - being here, it looks like it's within the scope of `MultiSelectEntry` when really it's updating the global variable `ko.bindingHandlers`. As is, it's updating that global every time a multi select question is created, which probably isn't breaking anything but is rather weird. The only other custom binding in web apps is [here](https://github.com/dimagi/commcare-hq/blob/ea29aa52187e2d49e9da9b54b0ba0e1624fffbab/corehq/apps/cloudcare/static/cloudcare/js/form_entry/form_ui.js#L27), so that's one place this could go.

Moving it out of here would mean you wouldn't have access to `self` anymore, so you'd need to pass more info as the binding's value. Maybe `MultiSelectEntry` could have a computed observable that returns the number of choices is `hideLabel` is true, and a flag like -1 if `hideLabel` is false, and you could pass that observable as the value? Or there could be a computed observable that just calculates the final class name, or blank if `hideLabel` is false, and then this binding would be a generic way to add a dynamic class. What do you think?

Reply:
Luckily the custom binding is gone so we don't have to worry about this! Thought they were well-thought 
contingencies though.